.gitignore

```diff
@@ -1,6 +1,7 @@
 node_modules
 npm-debug.log
 yarn-error.log
+.log
```

Comment:
what generates this file ?

Reply:
I was playing with https://github.com/theia-ide/typescript-language-server which generates logs